pkg/api/latest/latest.go

```diff
@@ -122,9 +123,15 @@ func init() {
 		"Namespace": true,
 	}
 
+	// these kidns should be excluded from the list of resources
+	ignoredKinds := util.NewStringSet("ListOptions", "DeleteOptions", "Status", "ContainerManifest")
```

Comment:
typo: kidns

More importantly, ignored in what context? We currently have both master and kubelet resources defined in the types.go files.